external_file_location/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+External File Location
```

Comment:
Use last version of template
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst